app/models/ssg_revision.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# SSG config file revisions from compliance-ssg
+class SsgRevision < ApplicationRecord
```

Comment:
I see that we've chosen the rows option (vs. a single row). If we are planning to store more here, we could name it more generally ("SSG" abbreviation is all over the place :smile:). Smth like `ImportedRevisions` or just `Revisions`. Just an idea.

Reply:
:+1: Revisions